contents/ja/tour/hello_world.html

```diff
@@ -1,4 +1,4 @@
-<h1>ハローワールド !</h1>
+<h1>ハローワールド!</h1>
 <p>最初の Ebitengine のゲームとして、「 Hello, World! 」と表示してみましょう !</p>
```

Comment:
Remove the spaces in `「 Hello, World! 」` in the same way.

Also, remove all the spaces before `!` in this page.